### PR TITLE
chore: add issue template & auto stale issues and pr

### PR DESCRIPTION
**Pull request by imbajin**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+assignees: ''
+
+---
+
+<details>
+<summary> Note ( 特别注意, 点击展开 ) </summary>
+
+### Note ( 特别注意 ) : 
+
+> 1. 请先**搜索**, 并**确认**现有的 [Issues](https://github.com/hugegraph/hugegraph/issues) 与 [FAQ](https://hugegraph.github.io/hugegraph-doc/guides/faq.html) 中没有与您相同 / 相关的问题, 请勿重复提交
```
Comment (zhoney) [inline .github/ISSUE_TEMPLATE/bug_report.md]: tools issues link

File: .github/ISSUE_TEMPLATE/question_ask.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+assignees: ''
+
+---
+
+<details>
+<summary> Note ( 特别注意, 点击展开 ) </summary>
+
+### Note ( 特别注意 ) : 
+
+> 1. 请先**搜索**, 并**确认**现有的 [Issues](https://github.com/hugegraph/hugegraph/issues) 与 [FAQ](https://hugegraph.github.io/hugegraph-doc/guides/faq.html) 中没有与您相同 / 相关的问题, 请勿重复提交
```
Comment (zhoney) [inline .github/ISSUE_TEMPLATE/question_ask.md]: ditto

**Review by zhoney — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (imbajin) [reply .github/ISSUE_TEMPLATE/bug_report.md]: notice it, consider server-issue-page include many issues of outside module (like `loader/tool/hubble/client`)

(before we transfer all old issues properly) maybe users need search both of them, so I keep the url for server now

(or any suggestion?)

Comment (zhoney) [reply .github/ISSUE_TEMPLATE/bug_report.md]: prefer to add both server-issue-page and tools-issue-page now.

Comment (imbajin) [reply .github/ISSUE_TEMPLATE/bug_report.md]: replace FAQ by current issue link now (so as others modules)

**Review by zhoney — APPROVED** [review]

**Review by Linary — APPROVED** [review]

**Pull request merged**